### DISCUSSION TITLE
NETSDK1147 To build this project, the following workloads must be installed: wasm-tools

### DISCUSSION THREAD

**Discussion by MarkBrainfire** [unoplatform/uno#14191] [General]
I'm building an Uno Platform app taking advantage of the new MAUI 3rd party control "Islands".  
My target platforms are Windows and Android.
It was working fine, but now when attempting to Clean and Rebuild, Clean fails saying I need to install the "wasm-tools" workload??
I'm not even targeting WASM.  Everything worked fine before, so what has changed?
I updated and ran Uno-Check.
I went to the project directory and ran "dotnet workloads restore"
From the command line I've also run:
dotnet worklload install wasm-tools
dotnet worklload install wasm-tools-net6
dotnet worklload install wasm-tools-net7

Nothing works. When I create a brand-new Uno Platform App project targeting only Windows and Android, I get the same error.

How do I install the "wasm-tools" workload so that I can successfully compile my project again??

Visual Studio 17.7.6
.NET 7

**Comment by MarkBrainfire** [comment]
UPDATE: Using the preview version of Visual Studio (17.8.0 Preview 6.0) resolves the issue, which remains when using VS 17.7.6

**Reply by jeromelaban** [reply]
See https://github.com/unoplatform/uno/discussions/14268#discussioncomment-7477142 on how to apply a possible workaround with .NET 8's `dotnet workload clean` command.
